src/test/java/org/gridsuite/study/server/SensitivityAnalysisTest.java

```diff
@@ -731,4 +736,24 @@ public void testSensitivityAnalysisParameters() throws Exception {
                 status().isOk(),
                 content().string(SENSITIVITY_ANALYSIS_UPDATED_PARAMETERS_JSON));
     }
+
+    @Test
+    public void testGetComputationCount() throws Exception {
+        StudyEntity studyEntity = insertDummyStudy(UUID.randomUUID(), UUID.randomUUID());
+        UUID studyNameUserIdUuid = studyEntity.getId();
+        String mnBodyJson = objectWriter.writeValueAsString(IDS);
+        MvcResult mvcResult;
+        String resultAsString;
+        mvcResult = mockMvc.perform(
+                post("/v1/studies/{studyUuid}/sensitivity-analysis/computation-count", studyNameUserIdUuid)
+                        .header("userId", "userId")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(mnBodyJson)).andExpect(status().isOk()).andReturn();
+        resultAsString = mvcResult.getResponse().getContentAsString();
+        String count = mapper.readValue(resultAsString, String.class);
```

Comment:
assertEquals("4", resultAsString );    is enough

Reply:
done